stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/message/composer/MessageComposerView.kt

```diff
@@ -31,10 +39,20 @@ public class MessageComposerView : ConstraintLayout {
     public var onInputChangedHandler: (String) -> Unit = {}
 
     /**
-    * Callback invoked when clear button is clicked
-    */
+     * Callback invoked when clear button is clicked
+     */
     public var onDismissMessageHandler: () -> Unit = {}
 
+    /**
+     * Callback invoked when attachments were selected
```

Comment:
I guess `are selected`

Reply:
We also have this issue of sometimes calling these handlers, sometimes callbacks.

Maybe not having anything at all makes sense? 

E.g. `onMessageDismiss/ed` or something?

Reply:
Changing to `onMessageDismissed` style